images/prow-tests/Dockerfile

```diff
@@ -12,13 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/k8s-testimages/kubekins-e2e:v20200827-e0bb92b-master
-
-# Install extras on top of base image
+FROM gcr.io/k8s-testimages/kubekins-e2e:v20200827-e0bb92b-master AS stable-stuff
 
 ENV DEBIAN_FRONTEND noninteractive
+# If we don't run update, the apt-get install below don't work
 RUN apt-get update
-RUN gcloud components update
+
+# TODO: do we have to do this?
+# https://github.com/kubernetes/test-infra/blob/master/images/bootstrap/Dockerfile#L67
+# Is where it is installed, using rapid channel
+# RUN gcloud components update
```

Comment:
We have it because sometimes we might want to update the gcloud version without updating the Dockerfile.

Reply:
So add your response as a comment and keep the line commented out?

Reply:
SGTM